examples/with-xata/pages/_app.tsx

```diff
@@ -0,0 +1,62 @@
+import type { AppProps } from 'next/app'
+import Head from 'next/head'
+import '../styles/root.css'
+
+const metas = {
+  title: 'Next.js with-xata',
+  description: 'Run Next.js with Xata with this awesome template',
+  image:
+    process.env.NODE_ENV === 'development'
```

Comment:
Why is this needed?

Reply:
because OG:Image requires it to be absolute urls, pointing it to localhost on dev makes it easier to test a new image, I thought it was a nice addition to whoever bases on this template to build something.

But absolutely not _needed_